code/script.js

```diff
@@ -1,51 +1,177 @@
+};
+
+//Bot asks wich kind of activity
+  const activityQuestion = () => {
+    showMessage(`Nice to meet you ${yourName}! Which activity do you want to book?`, 'bot');
+    inputWrapper.innerHTML = //add three activity buttons
+    `<button id="kayak-btn" type="submit">Kayaking</button>
+    <button id="dive-btn" type="submit">Diving</button>
+    <button id="climb-btn" type="submit">Climbing</button>`;
+    //activity buttons should show up here
+  document.getElementById('kayak-btn')
+.addEventListener('click', () => handleActivityQuestion('Kayaking'));
+document.getElementById('dive-btn')
+.addEventListener('click', () => handleActivityQuestion('Diving'));
+document.getElementById('climb-btn')
+.addEventListener('click', () => handleActivityQuestion('Climbing'));
+   };
+
+  
+//Bot asks where the activity should take place
+const whereQuestion = () => {
+    showMessage(`Where do you want to go on your adventure?`, 'bot');
+inputWrapper.innerHTML = //add three where buttons
+`<button id="bali-btn" type="submit">Bali</button>
+<button id="iceland-btn" type="submit">Iceland</button>
+<button id="hawaii-btn" type="submit">Hawaii</button>`;
+//where buttons should show up here
+document.getElementById('bali-btn')
+.addEventListener('click', () => handleWhereQuestion('Bali'));
+document.getElementById('iceland-btn')
+.addEventListener('click', () => handleWhereQuestion('Iceland'));
+document.getElementById('hawaii-btn')
+.addEventListener('click', () => handleWhereQuestion('Hawaii'));
+};
+
+//OLD
+//Orderconfirmation. last message yes or no
+// const orderConfirmation = () => {
+//   showMessage(`Greate choice, Would you like to make a booking?`, "bot");
+//   inputWrapper.innerHTML = //add two choice buttons
+//   `<button id="yes" value="yes">Yes please!</button>
+//   <button id="no" value="no">No thanks!</button>`;
+// //yes or no buttons should show here
+//   document.getElementById('yes')
+//   .addEventListener('click', () => handleOrderConfirmation('Yes please!'));
+//   document.getElementById('no')
+//   .addEventListener('click', () => handleOrderConfirmation('No thanks!'));
+// };
+
+const orderConfirmation = () => {
+  showMessage(`Great choice, Would you like to make a booking?`, "bot");
+  inputWrapper.innerHTML = 
+  `<button id="yes" value="yes">Yes please!</button>
+  <button id="no" value="no">No thanks!</button>`;
+
+  document.getElementById('yes')
+  .addEventListener('click', () => {
+    handleOrderConfirmation('Yes please!');
+    showMessage(`Awesome! Your booking has been confirmed!`, "bot");
+  });
+
+  document.getElementById('no')
+  .addEventListener('click', () => {
+    handleOrderConfirmation('No thanks!');
+    showMessage(`No problem, let me know if you change your mind.`, "bot");
+  });
+};
+
+
+//Thank you! Have a great day!!
+const lastMessage = () => {
+    showMessage(`Thank you, have a great day!`, 'bot');
+  inputWrapper.innerHTML =``;
+};
+
+const handleInput = (event) => {
```

Comment:
The way your bot is structured, you do not really need this function. You're only calling it once, after the greetUser-function, and from then on your other functions are just calling the next one manually.

If you wanted to use this, you should put questionIndex++; in the start of every other function, and make all the click-events in the addEventListeners call handleInput instead. That way, the questionIndex-variable is increasing by 1 for every function it goes through, and the if-loop in handleInput will take care of the rest.

I am not saying you HAVE to do this as the way you are doing it works fine and is actually easier to read and understand! :)